miden-lib/src/tests/test_prologue.rs

```diff
@@ -1,7 +1,7 @@
 use miden_objects::{
     assembly::ProgramAst,
     transaction::{PreparedTransaction, TransactionArgs, TransactionScript},
-    utils::collections::*,
+    utils::{BTreeMap, Vec},
```

Comment:
Let's import these from `alloc`.

Also, we should probably remove these from `utils` module.